bridge-deploy/bridge_deploy/cli.py

```diff
@@ -0,0 +1,276 @@
+import click
+
+# from web3 import Web3, EthereumTesterProvider
+
+from deploy_tools.cli import (
+    jsonrpc_option,
+    keystore_option,
+    gas_option,
+    gas_price_option,
+    nonce_option,
+    auto_nonce_option,
+    connect_to_json_rpc,
+    retrieve_private_key,
+    get_nonce,
+)
+from deploy_tools.deploy import build_transaction_options
+from deploy_tools.files import validate_and_format_address, InvalidAddressException
+
+from bridge_deploy.home import (
+    deploy_home_bridge_contract,
+    initialize_home_bridge_contract,
+)
+
+
+def validate_address(ctx, param, value):
+    # TODO:
+    # Potentially reformat this to deploy-tools? (Copypasta from validator-set-deploy)
+    # Refactoring might be a thing.
+    """This function must be at the top of click commands using it"""
+    try:
+        return validate_and_format_address(value)
+    except InvalidAddressException as e:
+        raise click.BadParameter(
+            f"The address parameter is not recognized to be an address: {value}"
+        ) from e
+
+
+validator_set_address_option = click.option(
+    "--validator-set-address",
+    help='The address of the validator set contract, "0x" prefixed string',
```

Comment:
Is it? 
I thought the address given here should be either the address of the proxy or of the bridge validators (probably the proxy).

Reply:
Could be both. Clarified description.